### ISSUE TITLE
# {set_name}.tsv

### ISSUE THREAD

**Issue by Pwoer-zy** [Jason-Qiu/MultiSum_model#1]
After preprocessing, there should be 2 sets of training, validation, test sets. {set_name}.tsv is for the whole sentence and video environment and {set_name}_2.tsv is for the segmented video environment.

Where can I download {set_name}.tsv to download? Are you convenient to provide a link? Thank you so much

**Comment by willxxy** [comment]
Hello,

Thank you for your interest in our work! We have uploaded the whole sentence tsv files at this link under the [features folder](https://drive.google.com/drive/folders/1ZE3p7JmoZe0EK-HIxpKrYUdHqXwFabUf?usp=sharing). 

Thank you,
William Han

**Comment by lousiming** [comment]
> After preprocessing, there should be 2 sets of training, validation, test sets. {set_name}.tsv is for the whole sentence and video environment and {set_name}_2.tsv is for the segmented video environment.
> 
> Where can I download {set_name}.tsv to download? Are you convenient to provide a link? Thank you so much

Could you let me know if you downloaded the original video? If you did, could you please send me a message? It would be very helpful for me, and I hope you can assist me with this.

**Comment by lousiming** [comment]
@willxxy 
I am a graduate student from China, and I am very interested in your work on the MMSum dataset.

While working with the dataset, I encountered some issues. Specifically, I have been trying to download the YouTube videos using the links provided in the annotation files. However, I noticed that in some subcategories, at least 2-3 videos are no longer available.

I wanted to ask if you might have already downloaded these videos. If so, would it be possible to share a Google Drive link or any other method for me to access the missing videos? I sincerely request your assistance, as this would be incredibly helpful for my research.

Thank you very much for your time and support!
